_data/projects/Polymorph.yml

```diff
@@ -0,0 +1,11 @@
+name: Polymorph
+desc: Ultra-flexible self-hostable organisation tool (and so much more)
+site: acenturyandabit.github.io/polymorph
```

Comment:
@acenturyandabit Please add complete URL with http/https otherwise Travis will fail for this PR.

Reply:
gotcha, done.